src/Microsoft.ML.StandardTrainers/Standard/Online/AveragedPerceptron.cs

```diff
@@ -30,7 +30,7 @@ namespace Microsoft.ML.Trainers
     /// To create this trainer, use [AveragedPerceptron](xref:Microsoft.ML.StandardTrainersCatalog.AveragedPerceptron(Microsoft.ML.BinaryClassificationCatalog.BinaryClassificationTrainers,System.String,System.String,Microsoft.ML.Trainers.IClassificationLoss,System.Single,System.Boolean,System.Single,System.Int32))
     /// or [AveragedPerceptron(Options)](xref:Microsoft.ML.StandardTrainersCatalog.AveragedPerceptron(Microsoft.ML.BinaryClassificationCatalog.BinaryClassificationTrainers,Microsoft.ML.Trainers.AveragedPerceptronTrainer.Options)).
     ///
-    /// [!include[io](~/../docs/samples/docs/api-reference/io-columns-binary-classification.md)]
+    /// [!include[io](~/../docs/samples/docs/api-reference/io-columns-binary-classification-no-prob.md)]
```

Comment:
>no-prob [](start = 92, length = 7)

I think SDCA also has a non-calibrated version, doesn't it?

Reply:
And it looks like `FastForest` doesn't calibrate either: `Fit` returns `BinaryPredictionTransformer<FastForestBinaryModelParameters>`.

---
In reply to: [277884116](https://github.com/dotnet/machinelearning/pull/3551#discussion_r277884116) [](ancestors = 277884116)

Reply:
Sdca is just fixed in the 3rd iteration. FastForest can naturally produce probability.

---
In reply to: [277885158](https://github.com/dotnet/machinelearning/pull/3551#discussion_r277885158) [](ancestors = 277885158,277884116)